modules/schematics/src/action/index.spec.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
```

Comment:
Is this copyright declaration intentional?

Reply:
Some of the schematics were created from examples in the Angular schematics so I wasn't sure whether to remove them all or leave them.